geoportal/geoportailv3_geoportal/static-ngeo/js/search/SearchController.js

```diff
@@ -831,14 +831,22 @@ exports.prototype.setBackgroundLayer_ = function(input) {
 
 
 /**
- * @param {(Object|string|number)} input The input.
- * @param {string | undefined} key The key.
+ * @param {string} layerId
  * @private
  */
-exports.prototype.addLayerToStore_ = function(input) {
-  useLayers().toggleLayer(input, true, false)
+exports.prototype.addLayerToStore_ = function(layerId) {
+  useLayers().toggleLayer(layerId, true, false)
 }
 
+// -------------
+// v4 migration: this function is deprecated,
+// use addLayerToStore_(layerToAdd.id) instead
+// -------------
+/**
+ * @param {(Object|string|number)} input The input.
+ * @param {string | undefined} key The key.
+ * @private
+ */
 exports.prototype.addLayerToMap_ = function(input, key) {
```

Comment:
Maybe we should comment this method completely to be more obvious that it's not used anymore!?